lib/Gfx/TUM_Draw.c

```diff
@@ -808,6 +813,30 @@ int vInitDrawing(char *path) // Should be called from the Thread running main()
 
 	atexit(SDL_Quit);
 
+
+    //TEST
+    FILE *my_img = fopen("../resources/freertos.jpg", "r");
```

Comment:
@PhilippvK  I think this sort of process will be the best caching I can do. I will just need to rerun `SDL_CreateTextureFromSurface` for each image every time the context is rebound.